docs/specs/legacy.yml

```diff
@@ -134,35 +146,48 @@ outputs:
         "title": "Title",
         "rawTitle": "<h1 id=\"title\">Title</h1>",
         "_op_canonicalUrlPrefix": "https://docs.com/en-us/rawmetadata/",
-        "layout": "Conceptual",
         "document_id": "fdb5a0d7-b68e-c465-9c5e-54a5cda94902",
         "document_version_independent_id": "92a5a770-f05f-b8bb-81c9-a8d786a9e608",
         "_op_gitContributorInformation": {
-          "contributors": [],
+          "contributors": [{
+            "display_name": "Charlie",
+            "id": "3"
+          }],
           "author": {
-            "display_name": "docfx",
-            "id": "docfx@microsoft.com",
-            "profile_url": null
+            "display_name": "Bob",
+            "id": "2",
           }
         },
         "content_git_url": "https://github.com/a/rawmetadata/blob/master/c.md",
-        "gitcommit": "https://github.com/a/rawmetadata/blob/ff3949c8ae50dd622cd7c67952c8b020c93636a6/c.md",
+        "gitcommit": "https://github.com/a/rawmetadata/blob/760e36bf5dcf238e28e1eced0c98a6e9d9e444a8/c.md",
         "original_content_git_url": "https://github.com/a/rawmetadata/blob/master/c.md",
         "original_content_git_url_template": "{repo}/blob/{branch}/c.md",
         "!metadata": null,
         "!contributors": null,
-        "!bilingual": null,
+        "!enable_loc_sxs": null,
         "!content": null,
         "!schema_type": null,
-        "!author_info": null
+        "!author_info": null,
+        "!monikers": null,
       },
       {
         "url": "/rawmetadata/d.png"
       },
       {
         "url": "/rawmetadata/",
         "document_id": "b3c1f04b-5bdb-1c10-5237-024cf382b4cc",
-        "_path": "index.json"
+        "_path": "index.json",
+        "_op_gitContributorInformation": {
+          "update_at": "10/30/2018",
+          "updated_at_date_time": "2018-10-30T00:00:00Z",
+          "author": {
+            "name": "charlie",
+            "profile_url": "https://github.com/charlie",
+            "display_name": "Charlie",
+            "id": "3"
+          },
+          "!contributors": null
```

Comment:
that's a break change, will break template logic, any idea? @yufeih  @live1206 

![image](https://user-images.githubusercontent.com/19990166/56276087-7aec2200-6134-11e9-9f54-a12e7f5438ac.png)

Reply:
https://ceapex.visualstudio.com/Engineering/_git/docs-ui/pullrequest/10814?_a=overview has been created to fix it in template.